src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -237,7 +235,7 @@ private function createAccessControl($entityClass, $id, $permissions, Role $role
         return $accessControl;
     }
 
-    public function findByProvider()
+    public static function findByProvider(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes